### PR TITLE
Create a dmarc policy article

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dd13411bba/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-f925b9a290/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-25bbc4181c/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. My suggested edits are attached. Please let me know if you have any questions. Thanks!

[Create a DMARC Policy_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1192123/Create.a.DMARC.Policy_editsKH.pdf)
[Create a DMARC Policy_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1192124/Create.a.DMARC.Policy_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e1e40ad68a/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-288fbb1b05/how-to/). :bow:

File: content/rackspace-email/create-a-dmarc-policy.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+7. Click **Add Record**
+
+Your new settings take 24 to 48 hours to propagate. For more information on propagation, see [DNS propagation](/how-to/dns-record-definitions#dns-propagation). 
+
+### Select an aggregator
+
+Your DMARC policy is more valuable when you use an aggregator to help filter the
+content of the reports that are returned. Without an aggregator, the reports are in an
+XML format and are virtually unreadable. An aggregator formats this information and
+send out weekly reports to the email address specified, which contain the sending
```
Comment (catlook) [inline content/rackspace-email/create-a-dmarc-policy.md]: ... sends out ....
[might reorganize the sentence.  As it is, it looks like the which clause is modifying the specified email address, but I think it's meant to modify the weekly reports]

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b24c072a96/how-to/). :bow:

**Pull request merged**